workos/utils/request.py

```diff
@@ -9,7 +11,9 @@
 )
 
 BASE_HEADERS = {
-    "User-Agent": "WorkOS Python/{}".format(workos.__version__),
+    "User-Agent": "WorkOS Python SDK/{} Python/{}".format(
```

Comment:
It looks like in the Ruby SDK we swap the order here, showing the Python version first and then the SDK version.

Should we keep it consistent?

Reply:
I'm ok with both. No strong opinions. I'll push a commit. 👍